text/0119-instance-options.md

```diff
@@ -184,10 +184,10 @@ If no master key is provided in a `development` environment, all routes will be
 
 Deactivates Meilisearch's built-in telemetry collect when enabled.
 
-#### 3.3.6. Dumps destination
+#### 3.3.6. Dump destination
```

Comment:
Shouldn't we change for `Dump directory`?

Reply:
Yes absolutely, not sure how I missed that one!

```suggestion
#### 3.3.6. Dump directory
```